app/routes.js

```diff
@@ -75,7 +75,7 @@ router.use((req, res, next) => {
         !includes(config.whitelistedPagesAfterSubmission, req.originalUrl)
     ) {
         res.redirect('thankyou');
-    } else if (formdata.paymentPending === 'false' &&
+    } else if ((get(formdata, 'payment.total') === 0 || get(formdata, 'payment.status') === 'Success') &&
```

Comment:
It's more of a personal thing but to keep the code readable it might be better to assign (get(formdata, 'payment.total') === 0 || get(formdata, 'payment.status') === 'Success') to skipPayment flag and then include (skipPayment === 'false' in the expression. I think it's more readable that way.